interface/src/interfaces/Keys.js

```diff
@@ -0,0 +1,11 @@
+export const ARROW_UP = 'ARROW_UP'
```

Comment:
Stałki do klawiszy i akcji już gdzieś były definiowane, cnie? Wiesz, te w postaci binarnej. Nie moglibyśmy skorzystać z nich zamiast tworzyć nowe, stringowe? Bo to trochę powielanie tego co już mamy